Test/git-issues/git-issue-590.dfy

```diff
@@ -0,0 +1,3 @@
+// RUN: %dafny /compile:3 x.cs > "%t"
+// RUN: %diff "%s.expect" "%t"
+// XFAIL: *
```

Comment:
Is this test really an expected fail (XFAIL)? I would have guessed that line 1 causes an error status code (non-0) but writes output to the file `%t`. Then, line 2 compares that file with the `.expect` file, finds the two to be equal, and therefore reports a status code of success (0). If so, the test should be an expected pass.